PdCore/src/main/java/org/puredata/android/service/PdService.java

```diff
@@ -242,7 +241,7 @@ private Notification makeNotification(Intent intent, int icon, String title, Str
 		}
 
 		PendingIntent pi = PendingIntent.getActivity(getApplicationContext(), 0, intent, PendingIntent.FLAG_IMMUTABLE);
-		return new NotificationCompat.Builder(PdService.this, TAG)
+		return new Notification.Builder(PdService.this, TAG)
```

Comment:
What compatibility with old versions is being lost by this change?

Reply:
there again, my comprehension is limited...
All I know is javadoc runs successfully with this change, and apps are running normally.
But I believe this Notification thing needs more love anyway (actually I don't see the notification on devices...).

Reply:
By the way, here's where the javadoc errors are ignored in CI:

https://github.com/libpd/pd-for-android/blob/395ff52ab9aa7597847fd0da874967c4175600b0/PdCore/build.gradle#L109

Reply:
Of course!
Here I can run locally with `failOnError = true`